cmd/tt/main.go

```diff
@@ -58,6 +58,10 @@ var (
 			Key:      "thread",
 			DefValue: "",
 		},
+		"existing": {
```

Comment:
Should this live on in the bucket config?

Reply:
it's only used for determining how to create the local bucket, so it doesn't need to be saved anywhere after that. I tried just making it a normal flag not bound with viper, but had some problem. I can try that again.

Reply:
It should work if you put the flag definition _after_ calling _cmd.BindFlags_

Reply:
All good, I think previously I was getting the flag value from viper instead of `c`. All fixed now.